diff --git a/node_modules/bip32/src/bip32.js b/node_modules/bip32/src/bip32.js
index 66fd69a..8fec770 100644
--- a/node_modules/bip32/src/bip32.js
+++ b/node_modules/bip32/src/bip32.js
@@ -6,7 +6,7 @@ const bs58check = require('bs58check');
 const typeforce = require('typeforce');
 const wif = require('wif');
 function BIP32Factory(ecc) {
-    testecc_1.testEcc(ecc);
+    //testecc_1.testEcc(ecc);
     const UINT256_TYPE = typeforce.BufferN(32);
     const NETWORK_TYPE = typeforce.compile({
         wif: typeforce.UInt8,
